project/Build.scala

```diff
@@ -347,11 +349,39 @@ object ScaldingBuild extends Build {
     )
   ).dependsOn(scaldingCore)
 
+  lazy val scaldingParquetFixtures = module("parquet-fixtures")
+    .settings(ScroogeSBT.newSettings:_*)
+    .settings(
+      scroogeThriftSourceFolder in Test <<= baseDirectory {
+        base => base / "src/test/resources"
+      },
+      sourceGenerators in Test <+= (
+          streams,
+          scroogeThriftSources in Test,
+          scroogeIsDirty in Test,
+          sourceManaged
+      ).map { (out, sources, isDirty, outputDir) =>
+        // for some reason, sbt sometimes calls us multiple times, often with no source files.
+        if (isDirty && sources.nonEmpty) {
+          out.log.info("Generating scrooge thrift for %s ...".format(sources.mkString(", ")))
+          ScroogeSBT.compile(out.log, outputDir, sources.toSet, Set(), Map(), "java", Set("--language", "java"))
+        }
+        (outputDir ** "*.java").get.toSeq
+      },
+      libraryDependencies ++= Seq(
+        "com.twitter" %% "scrooge-serializer" % scroogeVersion % "provided",
+        "commons-lang" % "commons-lang" % "2.2",
+        "org.apache.thrift" % "libthrift" % thriftVersion
+      )
+    )
+
   lazy val scaldingParquet = module("parquet").settings(
     libraryDependencies <++= (scalaVersion) { scalaVersion => Seq(
+      "org.apache.parquet" % "parquet-column" % parquetVersion,
+      "org.apache.parquet" % "parquet-hadoop" % parquetVersion,
+      "org.apache.parquet" % "parquet-pig" % parquetVersion,
+      "org.apache.parquet" % "parquet-thrift" % parquetVersion
       // see https://issues.apache.org/jira/browse/PARQUET-143 for exclusions
-      "org.apache.parquet" % "parquet-cascading" % parquetVersion
-        exclude("org.apache.parquet", "parquet-pig")
```

Comment:
Added back the exclude. There was an unused pig import in the original code which we don't need.